db/schema.rb

```diff
@@ -22,27 +22,27 @@
     t.string   "job_title"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.integer  "veteran_id",   index: {name: "index_affiliations_on_veteran_id"}
+    t.integer  "veteran_id",   :index=>{:name=>"index_affiliations_on_veteran_id"}
   end
 
   create_table "awards", force: :cascade do |t|
     t.string   "title"
     t.string   "organization"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.integer  "veteran_id",   index: {name: "index_awards_on_veteran_id"}
+    t.integer  "veteran_id",   :index=>{:name=>"index_awards_on_veteran_id"}
```

Comment:
are these schema changes necessary? They look mostly syntactical. Were these changes the result of rubocop rule preferring hash-rocket?